### PR TITLE
Contact form validation

### PR DESCRIPTION
**Pull request by julzedz**
In this project, using javascript, we implemented a client-side validation of the data in the email input before submission using an event listener.

### CODE REVIEW

File: javascript/validate.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+const formElement = document.querySelector('#form-element');
+const email = formElement.elements.emaile;
+const error = document.querySelector('.error');
+const emailInput = email.value;
+const Emailregex = /^[a-z0-9]+@[a-z0-9-]+\.[a-z0-9-.]+$/;
+const errorMsg = 'Email should only contain lowercase letters.';
```
Comment (codecaiine) [inline javascript/validate.js:4]: - [ ] Good job so far but please make sure that the error message shows up close to the **submit** button to follow the project requirement :+1: 

Screenshot from 2022-09-09 13-50-40

File: javascript/validate.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+formElement.addEventListener('submit', (e) => {
```
Comment (codecaiine) [inline javascript/validate.js:7]: - [ ] From the requirement when the user submits the form, you check if the email is in lower case and If the validation is OK, the form is sent. In your case, we are getting an error message and the form is not sent. Please fix it to follow the requirements. :+1: 

Screenshot from 2022-09-09 13-51-12

**Review by codecaiine — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Descriptive PR :heavy_check_mark: 
- No linters errors :heavy_check_mark: 

## Required Changes ♻️
- [ ] Please make sure that you have the name of all the contributors in your readme to make it more professional :+1: 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -181,8 +181,9 @@ <h2>
       </header>
       <form id="form-element" action="https://formspree.io/f/meqdoagj" method="POST">
         <input class="input-field" type="text" name="name" maxlength="30" placeholder="Full name (required)" required>
-        <input class="input-field" type="email" name="email" placeholder="Email address (required)" required>
+        <input class="input-field" id="emaile" type="email" name="email" placeholder="Email address (required)" required>
         <textarea id="text-area" class="input-field" name="message" maxlength="500" placeholder="Enter text here (required)" required></textarea>
+        <small class="error">Error</small>
         <button id="form-button" type="submit">Get in touch</button>
       </form>
```
Comment (1GORDON) [inline index.html:188]: Great job @julzedz 🤝

- [ ] As the previous reviewer suggested, kindly make sure your form submits when the email is lowercase.
email

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @julzedz ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Jules, 
## Congratulations 🎉 Thank you for implementing all the requirements 🙏
### Please go ahead and merge the branch!

## Happy Coding 👏👏👏
#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**